tests/postgres/postgres.rs

```diff
@@ -2042,3 +2042,26 @@ async fn test_issue_3052() {
         "expected encode error, got {too_large_error:?}",
     );
 }
+
+#[sqlx_macros::test]
+async fn test_pg_copy_chunked() -> anyhow::Result<()> {
+    let mut conn = new::<Postgres>().await?;
+
+    // (1 GiB - 1) - command byte (1 byte) - length prefix (4 bytes)
+    const COPY_MAX_DATA_LEN: usize = 1_073_741_824 - 1 - 1 - 4;
```

Comment:
Instead of repeating this constant for the test, it could just be exported from the `postgres` module but `#[doc(hidden)]`.